xds/internal/server/listener_wrapper.go

```diff
@@ -1,3 +1,5 @@
+// +build !appengine
```

Comment:
Is this required? I have a PR out where I am getting rid of all these build constraints. The PR is just blocked because of test flakiness.

Reply:
Vet made me do this.